omrsigcompat/omrsig.cpp

```diff
@@ -362,9 +364,28 @@ omrsig_sigaction_internal(int signum, const struct sigaction *act, struct sigact
 		}
 
 		SIGLOCK(sigMutex);
-		/* Get previous handler from slot. */
 		if (NULL != oldact) {
-			*oldact = *savedAction;
+			struct sigaction oact = {{0}};
+			bool retOact = false;
+			rc = omrsig_signalOS_internal(signum, NULL, &oact);
+
+#if defined(POSIX_SIGNAL)
+			if (!(oact.sa_flags & SA_SIGINFO)) {
```

Comment:
Can you use the OMR_ALL_BITS_SET macros or explicitly check the `SA_SIGINFO == ....`?

Reply:
I searched the whole OMR repo, there is no OMR_ALL_BITS_SET.